README.md

```diff
@@ -1 +1,69 @@
-# sawtooth-mktplace
\ No newline at end of file
+# Sawtooth Marketplace
+
+_Sawtooth Marketplace_ is a blockchain application built on Hyperledger
+Sawtooth, allowing users to exchange quantities of customized "Assets" with
+other users on the blockchain. This repo contains a number of components which
+together with a _Hyperledger Sawtooth_ validator, will run a Sawtooth
+blockchain and provide a simple RESTful API to interact with it. The components
+in this repo will include:
+
+- a **rest api** which provides a REST API for querying blockchain data
+- a **transaction processor** which handles RBAC-specific transaction logic
+- a **ledger sync** which writes blockchain state changes to a local database
+- and an example **client** which uses Marketplace to create a loyalty point
+  program administered through a simple web app.
+
+## Usage
+
+The easiest way to run these components is with
+[Docker](https://www.docker.com/what-docker). To start these components,
+first install Docker for your platform and clone this repo.
+
+Then, generate Python versions of the Protobuf files for each component, by
+running the `protogen` command from the root project directory:
+
+```bash
+bin/protogen
```

Comment:
Should this be run in a docker container or what software needs to be installed on the host to do this?

Reply:
I'm not a big fan of spinning up docker containers to run one script. I also have not tracked down all the dependencies to run it locally. If we need to tidy up the developer tools, I think that should happen in a later PR, rather than holding up getting this structure into the repo.

Reply:
Absolutely everything needs to be done in docker containers, but I'd be fine with that in a separate PR.

Reply:
I have a PR built off of this one that adds a script to run protogen, lint, and unit tests in docker containers.

Reply:
#3 Adds scripts and updates the README.md for protogen in a docker container.